### PR TITLE
create signup page & login page 2.0.0

### PR DESCRIPTION
**Pull request by shivendra-webkul**
## Issue Reference
2.0.0

## Documentation
- [ ] My pull request requires an update on the documentation repository.

### CODE REVIEW

File: packages/Webkul/Shop/src/Resources/views/customers/sign-up.blade.php [hunk 1]
```diff
@@ -1,117 +1,164 @@
...
+						<a href="" class="bg-[position:-80px_-274px] bs-main-sprite w-[40px] h-[40px]"
+							aria-label="Pintrest"></a>
+						<a href="" class="bg-[position:-120px_-274px] bs-main-sprite w-[40px] h-[40px]"
+							aria-label="Linkdln"></a>
+						<a href="" class="bg-[position:0px_-314px] bs-main-sprite w-[40px] h-[40px]"
+							aria-label="Linkdln"></a>
+					</div>
+				</div>
+            </x-shop::form>
+			<p class="text-[#7D7D7D] font-medium mt-[20px]">@lang('shop::app.customer.signup-text.account_exists') <a class="text-navyBlue" href="{{ route('shop.customer.session.index') }}">@lang('shop::app.customer.signup-text.title')</a></p>
```
Comment (jitendra-webkul) [inline packages/Webkul/Shop/src/Resources/views/customers/sign-up.blade.php:160]: Add new line here

File: packages/Webkul/Shop/src/Resources/views/customers/sign-up.blade.php [hunk 2]
```diff
@@ -1,117 +1,164 @@
...
+		</div>
```
Comment (jitendra-webkul) [inline packages/Webkul/Shop/src/Resources/views/customers/sign-up.blade.php:161]: Add new line here

File: packages/Webkul/Shop/src/Http/Controllers/Customer/ResetPasswordController.php [hunk 3]
```diff
@@ -44,7 +36,7 @@ public function __construct(
      */
     public function create($token = null)
     {
-        return view($this->_config['view'])->with(
+        return view('shop::customers.reset-password')->with(
             ['token' => $token, 'email' => request('email')]
```
Comment (jitendra-webkul) [inline packages/Webkul/Shop/src/Http/Controllers/Customer/ResetPasswordController.php:40]: Break array into multiple lines

File: packages/Webkul/Shop/src/Resources/views/customers/sign-in.blade.php [hunk 4]
```diff
@@ -96,12 +121,36 @@ class="m-0 ml-[0px] block mx-auto w-full bg-navyBlue text-white text-[16px] max-
...
         </p>
     </div>
+
+    @push('scripts')
+
+        {!! Captcha::renderJS() !!}
+
+        <script>
+            function switchVisibility() {
+                var passwordField = document.getElementById("password");
```
Comment (devansh-webkul) [inline packages/Webkul/Shop/src/Resources/views/customers/sign-in.blade.php:145]: use let.

**Review by devansh-webkul — APPROVED** [review]

**Pull request merged**